README.md

```diff
@@ -218,6 +218,38 @@ lambda-cfn save dev
 
 We're happy you want to contribute! Check out [CONTRIBUTING.MD](CONTRIBUTING.MD) for more information.
 
+## Dispatch Integration
+
```

Comment:
I think we should add some more context here for people not familiar with Dispatch.

> `lambda-cfn` now supports the message router [Dispatch](https://github.com/mapbox/dispatch) which provides rich integration with Slack and GitHub. If the _optional_ `DispatchSnsArn` parameter is specified, `lambda-cfn` will grant the function permission to publish to the specified SNS Topic Arn. 

There should be something about how to use `lib/message.js` in here as well I think.